### PR TITLE
implement notes.md for competencies

### PR DESCRIPTION
**Pull request by shanioffe**

### CODE REVIEW

File: docs/competency-api/notes.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+### Example Curl request
+```bash
+curl -X PATCH \
+  -H "Content-Type: application/json" \
+  -H "Authorization": "Bearer Example.bearer.token" \
+  -d '{
+        notes: "This competency is meant for students with a Cyber Security track only."
+    }' \
+  -L "https://api.competency-constructor.clark.center/competencies/6112745b84804cf5833aa94c/behavior"
```
Comment (AStoerkel1) [inline docs/competency-api/notes.md]:
```suggestion
  -L "https://api.competency-constructor.clark.center/competencies/6112745b84804cf5833aa94c/notes"
```

**Review by AStoerkel1 — CHANGES_REQUESTED** [review]

File: docs/competency-api/notes.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+{
+    Authorization: "Bearer Example.bearer.token"
+}
+```
+
+### Request
+
+| Name | Required | Type | Description |
+|:----:|:-----:|:----:|-----|
+| notes | false | string | Any notes related to the competency |
```
Comment (JosueMolinaMorales) [inline docs/competency-api/notes.md]:
```suggestion
| details | true | string | Any notes related to the competency |
```
`notes` is actually `details` and it is required

File: docs/competency-api/notes.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+<details closed markdown="block">
+  <summary>
+    Example
+  </summary>
+
+### Example Http request body
+```json
+{
+    body: {
+        notes: "This competency is meant for students with a Cyber Security track only."
```
Comment (JosueMolinaMorales) [inline docs/competency-api/notes.md]:
```suggestion
        details: "This competency is meant for students with a Cyber Security track only."
```

File: docs/competency-api/notes.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+}
+```
+
+### Example Curl request
+```bash
+curl -X PATCH \
+  -H "Content-Type: application/json" \
+  -H "Authorization": "Bearer Example.bearer.token" \
+  -d '{
+        notes: "This competency is meant for students with a Cyber Security track only."
```
Comment (JosueMolinaMorales) [inline docs/competency-api/notes.md]:
```suggestion
        details: "This competency is meant for students with a Cyber Security track only."
```

**Review by JosueMolinaMorales — CHANGES_REQUESTED** [review]

**Review by pmanal1 — APPROVED** [review]

**Review by JosueMolinaMorales — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]

File: docs/competency-api/notes.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  -L "https://api.competency-constructor.clark.center/competencies/6112745b84804cf5833aa94c/notes"
+```
+</details>
+
+### Response
+
+| Name | Description |
+|:----:|----|
+| 200 | Notes updated successfully |
+| 400 | Competency is deprecated, rejected, or published and cannot be updated. Or body is incorrectly formatted. |
```
Comment (mfranz1) [inline docs/competency-api/notes.md]: I left this comment on Andy's and Paige's PRs but Can it be made more obvious that this is a list of 400 errors?

**Review by mfranz1 — CHANGES_REQUESTED** [review]

**Review by Cwagne17 — APPROVED** [review]

**Review by mfranz1 — CHANGES_REQUESTED** [review]
The competency.md I believe has it's own story. It should not be apart of this PR

**Review by mfranz1 — APPROVED** [review]

**Review by AStoerkel1 — APPROVED** [review]

**Pull request merged**